lib/bundler.rb

```diff
@@ -27,36 +27,36 @@ module Bundler
   autoload :DepProxy,               "bundler/dep_proxy"
   autoload :Deprecate,              "bundler/deprecate"
   autoload :Dsl,                    "bundler/dsl"
-  autoload :EndpointSpecification,  "bundler/endpoint_specification"
+  autoload :EndpointSpecification,  File.expand_path("bundler/endpoint_specification", __dir__)
```

Comment:
this seems... not right to me

Reply:
This is just an experiment not meant to stay in this PR, but I actually planned to open a separate PR to migrate our autoloads to require absolute paths. I think it would be a nice follow up to https://github.com/bundler/bundler/pull/7100 since we would make sure that we never use the $LOAD_PATH for requiring our own internal code.

What do you think it is not right?

By the way, it would be prettier if we had this: https://github.com/bundler/bundler/pull/7100 :smiley:.

Reply:
Marking this as solved because I removed the changes from this PR. I'll open a separate discussion for migrating our autoloads to not use the load path.